docs/make.jl

```diff
@@ -176,33 +184,37 @@ makedocs(sitename = "Oceananigans.jl",
          plugins = [bib],
          modules = [Oceananigans],
          warnonly = [:cross_references],
-         doctest = true, # set to false to speed things up
+         doctest = false, # set to false to speed things up
          clean = true,
          checkdocs = :exports) # set to :none to speed things up
 
-@info "Clean up temporary .jld2 and .nc output created by doctests or literated examples..."
-
 """
     recursive_find(directory, pattern)
 
 Return list of filepaths within `directory` that contains the `pattern::Regex`.
 """
-recursive_find(directory, pattern) =
-    mapreduce(vcat, walkdir(directory)) do (root, dirs, files)
-        joinpath.(root, filter(contains(pattern), files))
+function recursive_find(directory, pattern)
+    mapreduce(vcat, walkdir(directory)) do (root, dirs, filenames)
+        matched_filenames = filter(contains(pattern), filenames)
+        map(filename -> joinpath(root, filename), matched_filenames)
     end
+end
+
+@info "Cleaning up temporary .jld2 and .nc output created by doctests or literated examples..."
 
-files = []
 for pattern in [r"\.jld2", r"\.nc"]
-    global files = vcat(files, recursive_find(@__DIR__, pattern))
+    filenames = recursive_find(@__DIR__, pattern)
+
+    for filename in filenames
+        rm(filename)
+    end
```

Comment:
did you check that the changes you did here do what the previous code was doing?